### ISSUE TITLE
Update Solution Provider Process (And thus the Guide)

### ISSUE THREAD

**Issue by Mik-TF** [threefoldtech/info_grid#91]
This is a proposition for this part of the manual: https://github.com/threefoldtech/info_grid/blob/development/src/tfchain/tfchain_solution_provider.md

The solution provider topic has gotten a lot of attention and it has been clearly stated that we need to properly define what we ask of solution providers when they offer a solution.

The solution provider could mention the following:

* Type of solution: the solution provider should indicate the state of the code/solution:
  * open source
    * easily verifiable by TF
  * closed source
    * if closed source
      * the TF team should have access to the code to check that everything is done according to TF's ethos
* Marketing
  * The solution provider should indicate if they aim at promoting TF/the solution
    * this can help the solution provider to earn the maximum of the 50%
    * the solution provider should explain their marketing startegy/actions
* Utilization Rewards (up to 50%)
  * How much of the 50% utilization reward are they asking for? This should be mentioned in the proposition
    * For example, it has been proposed that offering the solution is one part, but doing marketing for TF/the solution is another part
      * In this regard, a solution provider proposing clear marketing goals and acts would be more likely to get the total 50%
    * Is it a new product offering or just an upgrade of an existing product?
      * e.g. a whole new product would merit to a greater extent the 50%

These are all points that would need to be discussed and agreed upon. That is why I brought them in this issue. This is not a final say on the topic.

A good example of a solution provider (IMO) that covers a lot of those grounds can be found here: https://forum.threefold.io/t/hostservice-nederland-solution-provider-request/3889

***

Bottom line: the clearer the indications for solution providers are, the better the program will be and the more we will attract and implement solutions.
